src/patternfly/components/Form/form.scss

```diff
@@ -76,89 +74,91 @@
       }
     }
   }
+}
 
-  .pf-c-form__group {
-    // Handle additional items within __group without having an additional wrapper or selector
-    > *:not(.pf-c-form__label):not(.pf-c-input-group):not(.pf-c-form-control):not(.pf-c-form__helper-text):not(.pf-c-form__legend):not(.pf-c-form__action-group):not(.pf-c-check):not(.pf-c-form__horizontal-group):not([class*="pf-l-"]) {
-      margin-left: var(--pf-c-form__group--MarginLeft);
-    }
+.pf-c-form__group {
+  // Handle additional items within __group without having an additional wrapper or selector
+  > *:not(.pf-c-form__label):not(.pf-c-input-group):not(.pf-c-form-control):not(.pf-c-form__helper-text):not(.pf-c-form__legend):not(.pf-c-form__action-group):not(.pf-c-check):not(.pf-c-form__horizontal-group):not([class*="pf-l-"]) {
+    margin-left: var(--pf-c-form--group--MarginLeft);
```

Comment:
can you update the variable at the top too :)

Reply:
oh oops!! I think it's this variable that's wrong actually since `.pf-c-form__group` is the selector. good catch :)

Reply:
nice! Did you notice the `margin-left` that gets applied to the `.pf-c-button` in the `.pf-c-form__action-group` ? It's coming from line 81. I think it's legacy from how the from groups used to be written. We could change it to `.pf-c-form__action .pf-c-button`. Although I'm questioning if we need line 81 at all now 🤔

Reply:
Hmm... I'm not seeing a left margin on a button in the form component or form demo on this branch or master. I also don't see `form-action-group.hbs` or `.pf-c-form__action-group` in use anywhere in the workspace, and that class isn't documented in the usage table, either. Is that an old class/element? I'm not sure what line 81 is supposed to target. Git blames you for adding it so I'll defer to your suggestion :) Let me know if you think we should get rid of it. I'm also happy to go over that or the whole file or whatever with you and we can clean it up.

Reply:
ya, it was legacy from when Matt Nolting did some work initially. I do remember wondering why it was there. I updated the action group to be a modifier so I probably just missed it. Take a look at my screenshot though, 
<img width="1439" alt="screen shot 2019-01-06 at 7 48 08 pm" src="https://user-images.githubusercontent.com/287880/50743267-13950c80-11ec-11e9-83bf-b5cfef407391.png">

I'd say we can pull it entirely.

Reply:
ahh! I was looking at the form component/demo action group examples. Agreed, doesn't look like it's necessary. Removed.